_format/1.1/index.md

```diff
@@ -162,10 +162,10 @@ possible for profiles to conflict with other profiles. Therefore, it is the
 responsibility of implementors to ensure that they do not support conflicting
 profiles.
 
-In the following example, a profile has specified that all timestamps use the
-[RFC 3339](https://tools.ietf.org/html/rfc3339) format and that all resource
-objects contain `created` and `modified` attributes. With such a profile
-applied, a response might appear as follows:
+In the following example, a profile has defined a `timestamps`
+[attribute][attributes] as an object that must contain both a `created` and
+`modified` member and that they must use the [RFC 3339](https://tools.ietf.org/html/rfc3339)
```

Comment:
@gabesullice this phrasing ("and that") is now awkward and no longer fits with the verb "has defined". Perhaps a second sentence or rephrasing?